platform/logging/LogLevel.ts

```diff
@@ -0,0 +1,70 @@
+import { assertNever } from '../lib/utils';
+
+/**
+ * Possible log level string values.
+ * @internal
```

Comment:
I've made `@internal` as much stuff as possible, it's easier to expose something later than remove it.